pkgs/applications/misc/ArchiSteamFarm/update.sh

```diff
@@ -1,10 +1,8 @@
 #!/usr/bin/env nix-shell
-#!nix-shell -I nixpkgs=../../../.. -i bash -p curl gnused jq common-updater-scripts
+#!nix-shell -i bash -p curl gnused jq common-updater-scripts
 set -euo pipefail
 cd "$(dirname "${BASH_SOURCE[0]}")"
 
-deps_file="$(realpath ./deps.nix)"
```

Comment:
Why was this removed? The call to the nuget deps generation script below is broken now.